web/app.fluidity.money/app/queries/useAirdropStats.ts

```diff
@@ -29,7 +29,10 @@ const queryAirdropStatsByAddress = gql`
 export const useAirdropStatsByAddress = async (address: string) => {
   const { url, headers } = fetchInternalEndpoint();
 
-  const variables = { address, now: new Date().toISOString() };
+  const variables = {
+    address: `0x${"0".repeat(24)}${address.slice(2)}`,
+    now: new Date().toISOString(),
```

Comment:
Would be better to compute the now date the moment the user presses the button and to hardcode the zero address as a constant, or to use ethers.constants.AddressZero